WooCommerce/WooCommerceTests/ViewRelated/OrderDetailsViewModelTests.swift

```diff
@@ -104,4 +105,16 @@ final class OrderDetailsViewModelTests: XCTestCase {
         let actionButtonIDs = viewModel.moreActionsButtons.map { $0.id }
         XCTAssertFalse(actionButtonIDs.contains(.editOrder))
     }
+
+    func test_paymentMethodsViewModel_title_contains_formatted_order_amount() {
+        // Given
+        let order = Order.fake().copy(currency: "EUR", total: "10.0")
+
+        // When
+        let currencyFormatter = CurrencyFormatter(currencySettings: .init())
+        let title = OrderDetailsViewModel(order: order, currencyFormatter: currencyFormatter).paymentMethodsViewModel.title
+
+        // Then
+        XCTAssertTrue(title.contains("\u{20AC}10.0"))
+    }
```

Comment:
Nit: Should we change this test to USD or CAD and assert `\u{0024}10.0` instead? This test should fail (but passes!) as the `currencyFormatter` should return the Euro sign at the end of the value rather than at the beginning: `10.0€`, but it doesn't.

I'll open a new issue to investigate this, as I could reproduce it all through the app with a Store set up to Euros.

Reply:
Hey @iamgabrielma thanks for reviewing the PR promptly! I think having the currency to the left of the value is expected in this test case, because it uses the default `CurrencySettings` in L114 here which has `CurrencyPosition.left`:

https://github.com/woocommerce/woocommerce-ios/blob/755e0af21807d0fc0d0069fcf232b0ac7066ecbb/WooFoundation/WooFoundation/Currency/CurrencySettings.swift#L22 

this position is a WC setting that a merchant can configure in core 🙂